spec/models/payroll/payment_csv_row_spec.rb

```diff
@@ -7,21 +7,25 @@
     let(:row) { CSV.parse(subject.to_s).first }
     let(:payment_award_amount) { BigDecimal("1234.56") }
     let(:payment) { create(:payment, award_amount: payment_award_amount, claims: claims) }
+    let(:personal_details) do
+      {
+        national_insurance_number: generate(:national_insurance_number),
```

Comment:
if we do it in the `payments factory, we may consider also being explicit with the TRN here too.

Reply:
I feel that putting TRN here might be misleading, as it could imply that the TRN needs to be consistent when in fact the TRN is neither validated for consistency nor used in the class that is being tested here. What do you think?

Reply:
We may not explicitly check/validate it in code, but does it better represent what the claims would look like in reality?

Reply:
> does it better represent what the claims would look like in reality?

Definitely, but I suppose that I didn't consider this to always be something worth aiming for in test data. i.e. that sometimes the only requirement of the test data for a unit test is that it be realistic enough for our class to be able to do its job. I'm happy to add it in, though.